src/tag/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2 on 2022-09-24 15:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Concept',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_date', models.DateTimeField(auto_now_add=True)),
+                ('updated_date', models.DateTimeField(auto_now=True)),
+                ('openalex_id', models.CharField(max_length=255)),
+                ('display_name', models.CharField(max_length=255)),
+                ('description', models.TextField(blank=True, default='')),
+                ('openalex_created_date', models.CharField(max_length=255)),
+                ('openalex_updated_date', models.CharField(max_length=255)),
```

Comment:
I don't think we should store the `openalex_created_date` and `openalex_updated_date` because they don't give us value (though I could be missing something) also, openalex concept may be updated on their end but the `openalex_updated_date` will not reflect this change because they do not push changes to us.

Reply:
i think it's OK to have this data, in case we ever want to do anything with it

Reply:
The value I was thinking of is mainly to see if our version of the concept is outdated by comparing against the same concept's updated_date as returned by the openalex api.

Also it may come in handy for debugging purposes later.